docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+   "metadata": {},
+   "source": [
+    "# T<sub>2</sub> Hahn Characterization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The purpose of the $T_2$ Hahn Echo experiment is to determine $T_2$ qubit property. \n",
+    "\n",
+    "In this experiment, we would like to get a more precise estimate of the qubit's decay time. $T_2$ represents the amount of time required for the transverse magnetization to fall to approximately 37% ($\\frac{1}{e}$) of its initial value.\n",
+    "\n",
+    "Since the qubit exposed to other noises (like $T_1$), we are using a $Rx(\\pi)$ pulse for decoupling and to solve our inaccuracy for the qubit frequency estimation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import qiskit\n",
+    "from qiskit.utils import apply_prefix\n",
+    "from qiskit_experiments.library.characterization.t2hahn import T2Hahn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The circuit used for the experiment comprises the following:\n",
+    "\n",
+    "    1. Rx gate\n",
+    "    2. delay\n",
+    "    3. measurement\n",
+    "\n",
+    "The user provides as input a series of delays and the time unit for the delays, e.g., seconds, milliseconds, etc. During the delay, we expect the qubit to precess about the z-axis. Because of the echo gate ($Rx(\\pi)$) for each echo, the angle will converge after the delay gates as following $\\theta_{new} = \\theta_{old} + \\pi$. By varying the extension of the delays, we get a series of decaying measurements. We can draw the graph of the resulting function and can analytically extract the desired values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set the computation units to microseconds\n",
+    "unit = \"us\"  # microseconds\n",
+    "qubit = 0\n",
+    "# set the desired delays\n",
+    "conversion_factor = 1e-6\n",
+    "delays = list(range(1, 50, 1) )\n",
+    "delays = [float(_) * conversion_factor for _ in delays]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "     ┌─────────┐┌─────────────────┐┌───────┐┌─────────────────┐┌─────────┐┌─┐\n",
+      "  q: ┤ Rx(π/2) ├┤ Delay(1e-06[s]) ├┤ Rx(π) ├┤ Delay(1e-06[s]) ├┤ Rx(π/2) ├┤M├\n",
+      "     └─────────┘└─────────────────┘└───────┘└─────────────────┘└─────────┘└╥┘\n",
+      "c: 1/══════════════════════════════════════════════════════════════════════╩═\n",
+      "                                                                           0 \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create a T2Hahn experiment. Print the first circuit as an example\n",
+    "exp1 = T2Hahn(qubit, delays)\n",
```

Comment:
Explicitly set the number of echoes, or at least explain that the default is 1

Reply:
Added a variable `number_of_echoes = 1` and i pass it to `T2Hahn` so it will be clearer.